mobile-app/app/screens/AppNavigator/screens/Balances/BalancesScreen.tsx

```diff
@@ -241,8 +252,63 @@ export function BalancesScreen ({ navigation }: Props): JSX.Element {
     }
   ]
 
+  // Asset sort bottom sheet list
+  const [assetSortType, setAssetSortType] = useState<BalancesSortType>(BalancesSortType.HighestDenominationValue) // to display selected sorted type text
+  const [isSorted, setIsSorted] = useState<boolean>(false) // to display acsending/descending icon
+  const [hideIcon, setHideIcon] = useState(false)
+  const [showAssetSortBottomSheet, setShowAssetSortBottomSheet] = useState(false)
+  const modifiedDenominationCurrency = useMemo(() => denominationCurrency === 'USDT' ? 'USD' : denominationCurrency, [denominationCurrency])
+  const sortTokensAssetOnType = useCallback((assetSortType: BalancesSortType): BalanceRowToken[] => {
+    console.log({ assetSortType })
+    switch (assetSortType) {
```

Comment:
Is it possible to just store the function into a variable. Then at the last step, we just use the sort?

Reply:
Updated!